docs/scripting/trust scripts/scripts.quine.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: scripts.quine
+---
+
+## Description
+
+scripts.quine outputs the source code of a player-made script.
+
+### Security Level
+
+FULLSEC
+
+## Syntax
+
+### CLI
+
+On the CLI, scripts.quine returns a Failure object and usage information.
+
+```
+>>scripts.quine
+Failure
+Use scripts.quine to output the source code of your script. Place the following into your script:
+return #fs.scripts.quine()
+```
+
+### Script
+
+```
+#fs.scripts.quine()
+```
+
+### Parameters
+
+No known parameters.
+
+### Return
+
+#### CLI
+
+On the CLI, scripts.quine returns an object.
+
+#### Script
+
+Called as a subscript, scripts.quine returns a string.
+
+```
+:)
```

Comment:
```suggestion
```

Isn't this a syntax error?

Reply:
I put it how the example would return to the screen, it seems to be passing the linter. I can change it to something that doesn't have the ")" or commit the suggestion i'm not picky

Reply:
This makes a lot more sense. I was this as code due to skimming. You can dismiss this change.